docs/tools/terraform/reference/cookbook/kafka-clickhouse-integration-recipe.rst

```diff
@@ -0,0 +1,163 @@
+    maintenance_window_dow  = "monday"
+    maintenance_window_time = "10:00:00"
+  }
+
+  resource "aiven_kafka_topic" "source" {
+    project      = var.project_name
+    service_name = aiven_kafka.kafka.service_name
+    partitions   = 2
+    replication  = 3
+    topic_name   = "iot_measurements"
+  }
+
+  resource "aiven_clickhouse" "clickhouse" {
+    project                 = var.project_name
+    cloud_name              = "google-europe-west1"
+    plan                    = "startup-beta-8"
+    service_name            = "clickhouse-gcp-eu"
+    maintenance_window_dow  = "monday"
+    maintenance_window_time = "10:00:00"
+  }
+
+  resource "aiven_service_integration" "clickhouse_kafka_source" {
+    project                  = var.project_name
+    integration_type         = "clickhouse_kafka"
+    source_service_name      = aiven_kafka.kafka.service_name
+    destination_service_name = aiven_clickhouse.clickhouse.service_name
+  }
+
+  resource "aiven_clickhouse_database" "measurements" {
+    project                 = var.project_name
+    service_name            = aiven_clickhouse.clickhouse.service_name
+    name                    = "iot_measurements"
+  }
+
+Execute the Terraform files
+'''''''''''''''''''''''''''
+
+.. dropdown:: Expand to check out how to execute the Terraform files.
+
+  1. Run the following command:
+
+    .. code-block:: shell
+
+       terraform init
+  
+  The ``init`` command performs initialization operations to prepare the working directory for use with Terraform. For this recipe, ``init`` automatically finds, downloads, and installs the necessary Aiven Terraform Provider plugins.
+
+  2. Run the following command:
+
+    .. code-block:: bash
+
+       terraform plan -var-file=var-values.tfvars
+  
+  The ``plan`` command creates an execution plan and shows the resources to be created (or modified). This command doesn't actually create any resources but gives you a heads-up on what's going to happen next.
+
+  3. If the output of ``terraform plan`` looks as expected, run the following command:
+
+    .. code-block:: bash
+
+       terraform apply -var-file=var-values.tfvars
+  
+  The ``terraform apply`` command creates (or modifies) your infrastructure resources.
+
+Check out the results
+---------------------
+
+* Resource ``aiven_clickhouse`` creates an Aiven for ClickHouse service with the project name, the cloud name (provider, region, zone), the Aiven service plan, and the service name as specified in the ``services.tf`` file.
+* Resource ``aiven_clickhouse_database`` resources creates a database for writing raw Kafka messages, where new tables and views are created based on the processed messages.
+* Resource ``aiven_kafka`` creates an Aiven for Apache Kafka cluster.
+* Resource ``aiven_kafka_topic`` creates Apache Kafka topic ``iot_measurements``.
+* Resource ``aiven_service_integration`` resource creates the integration between the Aiven for Apache Kafka and the Aiven for ClickHouse service.
```

Comment:
The servic integration creates a database to insert the ingested data to. In this instance, the database name will be ``service_kafka-gcp-eu`` (it depends on the kafka service name)